### PR TITLE
Update chrome browser for UI tests

### LINKED ISSUE #2743: Update chrome browser for UI tests
chrome browser was pinned to a working  version, need to find out why newer versions don't work and make them work

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Update the chrome browser to the latest version to run the accetance tests

## Related Issue

- Fixes #2743 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7488/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7492/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>XGAPortrait</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7492/

<p>





</p></details>

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js [hunk 1]
```diff
@@ -445,6 +445,7 @@ module.exports = {
       await this
         .useXpath()
         .getAttribute(linkSelector, 'filename', function (result) {
+          console.log(result)
```
Comment (kiranparajuli589) [inline tests/acceptance/pageObjects/FilesPageElement/filesList.js]: debug line!

**Comment by individual-it** [comment]
@jasson99 please follow up

File: tests/acceptance/setup.js [hunk 2]
```diff
@@ -34,6 +35,7 @@ Before(function createSessionForEnv () {
 })
 
 Before(function logSessionInfoOnSauceLabs () {
+  console.log(client.sessionId)
```
Comment (HariBhandari07) [inline tests/acceptance/setup.js]: debug?

File: tests/acceptance/setup.js [hunk 3]
```diff
@@ -110,6 +113,10 @@ After(function closeSessionForEnv () {
   return closeSession()
 })
 
+After(function rollbackConfigsOnLocal () {
+  return rollbackConfigs(client.globals.backend_url)
```
Comment (dpakach) [inline tests/acceptance/setup.js]: @jasson99  skip this part when running on ocis

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/pageObjects/FilesPageElement/filesList.js]: It was to reproduce the error and find where and why it actually fails.It is fixed now

File: tests/acceptance/setup.js (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/setup.js]: fixed now

File: tests/acceptance/setup.js (hunk 3 above)
Comment (jasson99) [reply tests/acceptance/setup.js]: done

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIOCIS</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/8951/

<p>

































































































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIOCIS</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/9047/

<p>













































































































































</p></details>

**Pull request merged**